src/main.ts

```diff
@@ -642,6 +650,9 @@ export async function mainAsync(params: GitParams | UserParams): Promise<void> {
                 if (replayScriptPath) {
                     await fs.promises.copyFile(replayScriptPath, path.join(resultDirPath, replayScriptFileName));
                 }
+                if (rawErrorsScriptPath) {
```

Comment:
I think we can just write it to its final location.

Reply:
We could, but it would require unnecessary gymnastics and this is consistent with how we handle the replay script.